### ISSUE TITLE
FastScroller?

### ISSUE THREAD

**Issue by csimmonsbluebook** [luizgrp/SectionedRecyclerViewAdapter#14]
Does anyone know of a fast scroller implementation that works with this library? The current fast scroller implementation I was using no longer works now that I am using this sectioning technique with the RecyclerView.

**Comment by sarpotdarsaurabh** [comment]
can we have this feature now..? Like A-Z index to the sides of the recyclerview...? Like Contacts app in android.?

**Comment by luizgrp** [comment]
Hi @sarpotdarsaurabh, 

This library is a custom adapter not a custom view. A Fastscroll isn't a feature for an adapter.

What have you tried so far and where did you get stuck? Which Fastscroll library have you tried?

A quick search on Google returned the following libs:
[RecyclerView-FastScroll](https://github.com/timusus/RecyclerView-FastScroll)
[FastScroll](https://github.com/L4Digital/FastScroll)
[recycler-fast-scroll](https://github.com/FutureMind/recycler-fast-scroll)

All of them you just have to make your adapter implement their interface. I don't see any problem in doing that with this library, but please let me know what you tried so far and what are the problems you faced.

**Comment by sarpotdarsaurabh** [comment]
hi @luizgrp ,
you are right that I will just have to implement their interface.  I realized it after I took a good look at the Fast scroll lib I am using which is [Alphabet-Fast-Scroll](https://github.com/myinnos/AlphabetIndex-Fast-Scroll-RecyclerView) this one. And I used your library in another project.
Extremely sorry for asking question without looking into lib code. Will implement the same interface as you suggested. If i face any problems will let you know. Basically what I need is Alphabetical sections (will achieve it using your lib) and index for those alphabets on the right. (will do it with that interface implementation now.)
Thank you for your prompt reply.
